cli/src/generate/grammar_files.rs

```diff
@@ -142,33 +142,28 @@ pub fn generate_grammar_files(
                     .unwrap();
                 if !dev_dependencies.contains_key("prebuildify") {
                     eprintln!("Adding prebuildify devDependency to package.json");
-                    dev_dependencies.insert("prebuildify".to_string(), "^6.0.0".into());
+                    dev_dependencies.insert("prebuildify".to_string(), "^6.0.1".into());
                     updated = true;
                 }
 
+                let node_test = "node --test bindings/node/*_test.js";
                 let scripts = package_json
                     .entry("scripts".to_string())
                     .or_insert_with(|| Value::Object(Map::new()))
                     .as_object_mut()
                     .unwrap();
-                match scripts.get("install") {
-                    None => {
-                        eprintln!("Adding an install script to package.json");
-                        scripts.insert("install".to_string(), "node-gyp-build".into());
-                        updated = true;
-                    }
-                    Some(Value::String(v)) if v != "node-gyp-build" => {
-                        eprintln!("Updating the install script in package.json");
-                        scripts.insert("install".to_string(), "node-gyp-build".into());
-                        updated = true;
-                    }
-                    Some(_) => {}
-                }
-                if !scripts.contains_key("prebuildify") {
-                    eprintln!("Adding a prebuildify script to package.json");
-                    scripts.insert(
-                        "prebuildify".to_string(),
-                        "prebuildify --napi --strip".into(),
+                if !scripts.get("test").is_some_and(|v| v == node_test) {
+                    eprintln!("Updating package.json scripts");
+                    scripts.clear();
+                    scripts.append(
+                        json!({
+                            "install": "node-gyp-build",
+                            "prestart": "tree-sitter build --wasm",
+                            "start": "tree-sitter playground",
+                            "test": node_test
+                        })
+                        .as_object_mut()
+                        .unwrap(),
```

Comment:
we can avoid the overhead of clearing

```suggestion
                    *scripts = Map::from_iter([
                        ("install".to_string(), "node-gyp-build".into()),
                        ("prestart".to_string(), "tree-sitter build --wasm".into()),
                        ("start".to_string(), "tree-sitter playground".into()),
                        ("test".to_string(), node_test.into()),
                    ]);
```